specs/wp-signup-spec.js

```diff
@@ -168,14 +188,70 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
+										stepNum++;
+
+										test.it( 'Can see a disabled publish button', function() {
+											const blogPostTitle = dataHelper.randomPhrase();
+											const blogPostQuote = dataHelper.randomPhrase();
+
+											this.adminTopBar = new AdminTopBar( driver );
+											this.adminTopBar.createNewPost();
+
+											this.editor = new EditorPage( driver );
+											this.editor.enterTitle( blogPostTitle );
+											this.editor.enterContent( blogPostQuote + '\n' );
+											return this.editor.publishEnabled().then( ( enabled ) => {
+												return assert.equal( enabled, false, 'Publish button is not disabled when activation link has not been clicked' );
+											} );
 										} );
 
-										test.it( 'Can see a single activation message', function() {
-											return this.emailClient.pollEmailsByRecipient( emailAddress ).then( function( emails ) {
-												return assert.equal( emails.length, 1, 'The number of invite emails is not equal to 1' );
+										test.it( 'Can see email verification required message', function() {
+											//This feature isn't currently working on mobile https://github.com/Automattic/wp-calypso/issues/10212
+											if ( screenSize === 'mobile' ) {
+												console.log( 'Email verification required messaging not supported on mobile - skipping test' );
+												return true;
+											}
+											return this.editor.emailVerificationNotice().then( ( displayed ) => {
+												return assert.equal( displayed, true, 'Email Verification Notice is not displayed when activation link has not been clicked' );
+											} );
+										} );
+
+										test.describe( `Step ${stepNum}: Can activate my account from an email`, function() {
+											stepNum++;
+
+											test.before( function() {
+												return this.emailClient = new EmailClient( signupInboxId );
+											} );
+
+											test.it( 'Can see a single activation message', function() {
+												return this.emailClient.pollEmailsByRecipient( emailAddress ).then( function( emails ) {
+													inboxEmails = emails;
+													return assert.equal( inboxEmails.length, 1, 'The number of invite emails is not equal to 1' );
+												} );
+											} );
+
+											test.describe( `Step ${stepNum}: Can publish when email is confirmed`, function() {
+												stepNum++;
+
+												test.it( 'Can not see a disabled publish button', function() {
+													driver.wait( until.elementTextContains( driver.findElement( webdriver.By.css( '.editor-ground-control__save-status' ) ), 'Saved' ), mochaTimeOut );
+													driver.get( inboxEmails[0].html.links[0].href );
+													driver.navigate().back();
+
+													this.editor = new EditorPage( driver );
+													const publishButton = driver.findElement( webdriver.By.css( '.editor-publish-button' ) );
+													driver.wait( until.elementIsEnabled( publishButton ), mochaTimeOut );
```

Comment:
The mochaTimeOut is the wrong timeout to use here, it should be the explicit wait timeout which is available in the page itself (see comment above)